### ISSUE TITLE
Binary Wheels Not Available for Python 3.13.0

### ISSUE THREAD

**Issue by mkmoisen** [confluentinc/confluent-kafka-python#1827]
Description
===========
Python 3.13.0 is available, however the binary wheels for confluent_kafka have not yet been built.

Would you please build and publish these wheels as soon as it is convenient for you?

How to reproduce
================

pip install confluent_kafka

Checklist
=========
Please provide the following information:

 - [x] confluent-kafka-python and librdkafka version (`confluent_kafka.version()` and `confluent_kafka.libversion()`):
 - [x] Apache Kafka broker version:
 - [x] Client configuration: `{...}`
 - [x] Operating system:
 - [x] Provide client logs (with `'debug': '..'` as necessary)
 - [x] Provide broker log excerpts
 - [x] Critical issue

**Comment by mkmoisen** [comment]
@pranavrth Would you kindly take a look? Thank you.

**Comment by pranavrth** [comment]
I am already working on it. Python 3.13 just got released today. Even [cibuildwheels](https://github.com/pypa/cibuildwheel) doesn't support it yet which we use to create wheels. We are planning to include Python 3.13 wheel in this release which is planned for 10/10.

**Comment by daneah** [comment]
A deeper challenge for this is that on macOS, when trying to build a wheel from source on Python 3.13.0, I get the following error:

<details>
<summary>Error</summary>

```
error: Failed to prepare distributions
  Caused by: Failed to fetch wheel: confluent-kafka==2.5.3
  Caused by: Build backend failed to build wheel through `build_wheel()` (exit status: 1)
--- stdout:
running bdist_wheel
running build
running build_py
copying src/confluent_kafka/error.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka
copying src/confluent_kafka/serializing_producer.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka
copying src/confluent_kafka/__init__.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka
copying src/confluent_kafka/deserializing_consumer.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka
copying src/confluent_kafka/schema_registry/avro.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/schema_registry
copying src/confluent_kafka/schema_registry/error.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/schema_registry
copying src/confluent_kafka/schema_registry/__init__.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/schema_registry
copying src/confluent_kafka/schema_registry/json_schema.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/schema_registry
copying src/confluent_kafka/schema_registry/schema_registry_client.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/schema_registry
copying src/confluent_kafka/schema_registry/protobuf.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/schema_registry
copying src/confluent_kafka/_util/__init__.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/_util
copying src/confluent_kafka/_util/conversion_util.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/_util
copying src/confluent_kafka/_util/validation_util.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/_util
copying src/confluent_kafka/serialization/__init__.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/serialization
copying src/confluent_kafka/admin/_metadata.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/admin
copying src/confluent_kafka/admin/_records.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/admin
copying src/confluent_kafka/admin/_acl.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/admin
copying src/confluent_kafka/admin/_resource.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/admin
copying src/confluent_kafka/admin/__init__.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/admin
copying src/confluent_kafka/admin/_cluster.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/admin
copying src/confluent_kafka/admin/_scram.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/admin
copying src/confluent_kafka/admin/_listoffsets.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/admin
copying src/confluent_kafka/admin/_config.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/admin
copying src/confluent_kafka/admin/_topic.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/admin
copying src/confluent_kafka/admin/_group.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/admin
copying src/confluent_kafka/avro/error.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/avro
copying src/confluent_kafka/avro/__init__.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/avro
copying src/confluent_kafka/avro/cached_schema_registry_client.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/avro
copying src/confluent_kafka/avro/load.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/avro
copying src/confluent_kafka/kafkatest/verifiable_client.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/kafkatest
copying src/confluent_kafka/kafkatest/__init__.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/kafkatest
copying src/confluent_kafka/kafkatest/verifiable_consumer.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/kafkatest
copying src/confluent_kafka/kafkatest/verifiable_producer.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/kafkatest
copying src/confluent_kafka/_model/__init__.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/_model
copying src/confluent_kafka/avro/serializer/__init__.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/avro/serializer
copying src/confluent_kafka/avro/serializer/message_serializer.py -> build/lib.macosx-11.0-arm64-cpython-313/confluent_kafka/avro/serializer
running build_ext
building 'confluent_kafka.cimpl' extension
clang -fno-strict-overflow -Wsign-compare -Wunreachable-code -DNDEBUG -g -O3 -Wall -arch arm64 -mmacosx-version-min=11.0 -Wno-nullability-completeness -Wno-expansion-to-defined -Wno-undef-prefix -isysroot /Applications/Xcode_15.2.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -fPIC -Werror=unguarded-availability-new -I/opt/homebrew/opt/openssl@3/include -I/opt/homebrew/opt/zlib/include -I/opt/homebrew/opt/sqlite/include -I/opt/homebrew/opt/libffi/include -I/opt/homebrew/opt/libpng/include -I/opt/homebrew/opt/zstd/include -I/Users/dhillard/Library/Caches/uv/builds-v0/.tmprQ5Yc2/include -I/Users/dhillard/.local/share/uv/python/cpython-3.13.0rc2-macos-aarch64-none/include/python3.13 -c /Users/dhillard/Library/Caches/uv/sdists-v4/index/371a95f244652c19/confluent-kafka/2.5.3/tmeJydwsuvBtbGD-z-X7V/confluent-kafka-2.5.3.tar.gz/src/confluent_kafka/src/Admin.c -o build/temp.macosx-11.0-arm64-cpython-313/Users/dhillard/Library/Caches/uv/sdists-v4/index/371a95f244652c19/confluent-kafka/2.5.3/tmeJydwsuvBtbGD-z-X7V/confluent-kafka-2.5.3.tar.gz/src/confluent_kafka/src/Admin.o
--- stderr:
Compiling with an SDK that doesn't seem to exist: /Applications/Xcode_15.2.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk
Please check your Xcode installation
Compiling with an SDK that doesn't seem to exist: /Applications/Xcode_15.2.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk
Please check your Xcode installation
clang: warning: no such sysroot directory: '/Applications/Xcode_15.2.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk' [-Wmissing-sysroot]
In file included from /Users/dhillard/Library/Caches/uv/sdists-v4/index/371a95f244652c19/confluent-kafka/2.5.3/tmeJydwsuvBtbGD-z-X7V/confluent-kafka-2.5.3.tar.gz/src/confluent_kafka/src/Admin.c:17:
In file included from /Users/dhillard/Library/Caches/uv/sdists-v4/index/371a95f244652c19/confluent-kafka/2.5.3/tmeJydwsuvBtbGD-z-X7V/confluent-kafka-2.5.3.tar.gz/src/confluent_kafka/src/confluent_kafka.h:18:
/Users/dhillard/.local/share/uv/python/cpython-3.13.0rc2-macos-aarch64-none/include/python3.13/Python.h:19:10: fatal error: 'assert.h' file not found
   19 | #include <assert.h>               // assert()
      |          ^~~~~~~~~~
1 error generated.
error: command '/usr/bin/clang' failed with exit code 1
```

</details>

It appears that there might be a hard-coded path expectation somewhere that is not met on some macOS configurations. On my system, there is no `/Applications/Xcode_15.2.app`—only `/Applications/Xcode.app`.

**Comment by daneah** [comment]
It looks like [cibuildwheel has added 3.13 support](https://github.com/pypa/cibuildwheel?tab=readme-ov-file#what-does-it-do).

**Comment by pranavrth** [comment]
Published [v2.6.0](https://github.com/confluentinc/confluent-kafka-python/releases/tag/v2.6.0) with [Python 3.13 wheels](https://pypi.org/project/confluent-kafka/). Closing this issue. Please raise a new issue if you face any problem.
